db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/ProcessInstanceWriter.java

```diff
@@ -33,12 +34,50 @@ public void create(final ProcessInstanceDbModel processInstance) {
             processInstance));
   }
 
-  public void update(final ProcessInstanceDbModel processInstance) {
-    executionQueue.executeInQueue(
-        new QueueItem(
-            ContextType.PROCESS_INSTANCE,
-            processInstance.processInstanceKey(),
-            "io.camunda.db.rdbms.sql.ProcessInstanceMapper.update",
-            processInstance));
+  public void end(
```

Comment:
I'm somehow not happy with the function name "end" :D

Reply:
💭  "Finish"? I couldn't use "complete" because it's already a state name (and there is also cancel/terminate).